### DISCUSSION TITLE
rustdesk-org / Octernships_Project

### DISCUSSION THREAD

**Discussion by anupam577** [rustdesk/rustdesk#4687] [General]
Elevate priviledge to run a Linux command with Rust
The starter code is included in this repository. ‼️

Setting up 🖥️
Prepare your Linux environment
Follow https://github.com/Desdaemon/flutter_rust_bridge_template/blob/main/README.md to get started
flutter run -d linux image
Task instructions
Your task is to

Elevate priviledge to run ls -la /root/ with Rust, and print the result on the Flutter window
Task Expectations
Polkit is usually used by priviledge elevation on Linux
Polkit may be disabled or uninstalled on some system, you need to detect if polkit works well, and elevate priviledge with alternative way

Could anyone explain more about the task?

**Comment by SergeyMy** [comment]
what is the purpose?
build rustdesk flutter version?

PS root privileges are definitely not needed!
